gossip.asciidoc

```diff
@@ -644,68 +644,388 @@ channel graph) has the following attributes:
+
+##### Anatomy of a Channel Update
+
+The channel update is the lifeblood of the core of the routing network.
+Advertising a new `channel_update` allows routing nodes to modify their fees,
+the status of their channel, security parameters, and other critical routing
+information.
+
+First, we'll enumerate each field, and provide a brief description. In a later
+section, we'll examine the fields that dictate a channel's directional routing
+policy in more detail.
+
+  * `signature`: A valid ECDSA signature over the `channel_update` message
+    digest signed by the node that's advertising this update.
+
+  * `chain_hash`: A 32-byte hash that denotes the chain that this channel is
+    anchored in. This is typically the genesis hash of the chain.
+
+  * `short_channel_id`: The 64-bit short channel ID that identifies which
+    channel in the Channel Graph is being updated.
+
+  * `timestamp`: A 32-bit time stamp typically interpreted as a unix epoch
+   timestamp which allows an update to create a relative sequencing of their
+   channel updates. An update with a higher timestamp is more recent than one
+   with a lower one.
+
+  * `message_flags`: A bitfield that's used to denotes the presence of optional
+   fields.
+
+  * `cltv_expiry_delta`: A forwarding related security parameter for the
+   channel. This value denotes the minimum gap (or delta) between the incoming
+   and outgoing HTLCs that are to be routed thorough this channel.
+
+  * `htlc_minimum_msat`: A value that denotes the smallest HTLC that's
+   permitted to be routed through this channel. This is expressed in
+   milli-satoshis.
+
+  * `fee_base_msat`: The _base_ fee that must be paid for all payments routed
+   through this node regardless of payment size. This is expressed in
+   milli-satoshis.
+
+  * `fee_proportional_millionths`: The _proportional_fee to be paid in
+   millionths of milli-satoshi. This is the denominator to be used in _fixed
+   point_ arithmetic when computing the _rate_ that needs to be paid when
+   routing through a channel.
+
+  * `htlc_maximum_msat`: The largest HTLC that is permitted to be routed
+   through this channel. This is expressed in milli-satoshis.
+
+Now that we know all the fields of the `channel_update` message, we'll describe
+at a high level how verification of a channel update is to be carried out. Most
+importantly, the `signature` MUST be valid, and the `timestamp` MUST always be
+increasing in order to invalidate prior updates.
+
+##### Channel & Message Flags
+
+As mentioned earlier, there're two `channel_update` messages for each direction
+of the channel. Therefore, we need to be able to identify which direction of
+the channel is being updated. If we have two nodes Alice, and Bob, then there
+may exist a channel update in the direction of Alice -> Bob, as well as for Bob
+-> Alice. In the first case, we require Alice to be the one that signs the
+update (as she will be carrying the HTLC), and in the second case, we require
```

Comment:
we have two different definitions of channel direction: 

1. DER encoding of lesser node public key
2. actual flow: Who offers an HTLCs. 

When I first learned about Lightning Networkwork I was very confused why the routing policy on a channel is defined by the node who adds the outgoing HTLCs and not why the routing fee is already for an incoming HTLC on my channel  

We mentioned implicitely here that there is this the direction with the outgoing HTLC. I think this should be defined somewhere very explicitely or appended with some info box doubeling down about the direction definitions

Reply:
I assume the "direction" of channels will already be mentioned in routing